nest-forms-backend/src/ginis/ginis.service.ts

```diff
@@ -427,12 +432,27 @@ export default class GinisService {
       return new Nack(false)
     }
 
-    if (!form.schemaVersion.pospID) {
+    const formDefinition = getFormDefinitionBySlug(form.formDefinitionSlug)
+    if (!formDefinition) {
+      throw this.throwerErrorGuard.NotFoundException(
+        FormsErrorsEnum.FORM_DEFINITION_NOT_FOUND,
+        `${FormsErrorsResponseEnum.FORM_DEFINITION_NOT_FOUND} ${form.formDefinitionSlug}`,
+      )
+    }
+    if (!isSlovenskoSkFormDefinition(formDefinition)) {
+      throw this.throwerErrorGuard.UnprocessableEntityException(
+        FormsErrorsEnum.FORM_DEFINITION_NOT_SUPPORTED_TYPE,
+        `onQueueConsumption: ${FormsErrorsResponseEnum.FORM_DEFINITION_NOT_SUPPORTED_TYPE}: ${formDefinition.type}, form id: ${form.id}`,
+      )
+    }
+
+    if (!formDefinition.pospID) {
```

Comment:
Not sure if this is needed, the type should be enough to cover this.

Reply:
Yes, true :) I will remove this.

Reply:
It should be removed now